### PR TITLE
Feature/api implemention

### LINKED ISSUE #8: API implemention
Create API interface to handle choregraphies and postures.

- [x] 1 endpoints : 1 type (advanced, basic move)
- [x] input: nom d une chore (pkl), number of chunks
- [x] output: chunk (slice audio, postures associees)

### PR DESCRIPTION
**Pull request by Anatole-DC**

### CODE REVIEW

File: koregraph/managers/choregraphy.py [hunk 1]
```diff
@@ -1,18 +1,11 @@
 from datetime import timedelta
-from pickle import load
+import pickle
```
Comment (Anatole-DC) [inline koregraph/managers/choregraphy.py]: Prefer `from module import function` as it is lighter.

If `load` is not clear enough as a function name, `... import load as load_pickle`.

File: koregraph/managers/choregraphy.py [hunk 2]
```diff
@@ -26,7 +19,7 @@ def load_choregraphy(name: str) -> Choregraphy:
     """
 
     with open(KEYPOINTS_DIRECTORY / f"{name}.pkl", "rb") as keypoints_file:
-        choregraphy_raw: Dict = load(keypoints_file)
+        choregraphy_raw: Dict = pickle.load(keypoints_file)
```
Comment (Anatole-DC) [inline koregraph/managers/choregraphy.py]: See first comment

File: koregraph/managers/choregraphy.py [hunk 3]
```diff
@@ -42,6 +35,24 @@ def load_choregraphy(name: str) -> Choregraphy:
...
+    Args:
+        data (Choregraphy): The choregraphy chunk
+        path (Path): The path to where we'll save the choregraphy chunk.
+
+    Returns:
+        Nothing
+    """
+    with open(path / f"{chore.name}.pkl", "wb") as handle:
+        pickle.dump(
+            {"keypoints2d": chore.keypoints2d, "timestamps": chore.timestamps},
```
Comment (Anatole-DC) [inline koregraph/managers/choregraphy.py]: See first comment

**Review by Kelly-an — DISMISSED** [review]

**Review by Kelly-an — APPROVED** [review]
All good

**Review by nonfinito — APPROVED** [review]
BRAVO

**Pull request merged**